src/component/Home.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from "react";
+import { EntryService } from "../service/EntryService";
+
+interface MyState {
+    loading: boolean;
+    message: string;
+}
+
+class Home extends React.Component<{},MyState> {
+    state: MyState = {
+      loading: true,
+      message: ''
+    };
```

Comment:
Dit kan in de constructor, wanneer je dit in de constructor doet dan snapt hij door type inference ook dat het om `{ loading: boolean, message: string }` gaat.